### PR TITLE
NISRA - Age based routing changes

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Age-based routing to be added for NISRA survey.

### How to review 
Ensure:
- Changes are as per instructions on Trello.

[Quick launch for NIR schema](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-NIR&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-3162-age-based-routing-nisra/data/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by MebinAbraham** [comment]
[Age based routing (NISRA)](https://trello.com/c/GW3UpPBO/3162-age-based-routing-nisra)

File: data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/arrive_in_country.jsonnet [hunk 1]
```diff
@@ -35,4 +35,24 @@ local proxyTitle = {
       when: [rules.proxyYes],
     },
   ],
+  routing_rules: [
+    {
+      goto: {
+        block: 'passports',
+        when: [
+          {
```
Comment (bambrp) [inline data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/arrive_in_country.jsonnet]:
```
         {
            id: 'country-of-birth-answer',
            condition: 'equals',
            value: 'Northern Ireland',
          },
```
You can't reach this block if you're from NI, so this rule (under 1s should skip one-year-ago questions) won't apply to anyone. The above block needs removing.

**Review by bambrp — CHANGES_REQUESTED** [review]

**Comment by MebinAbraham** [comment]
@bambrp @donnap573 Routing should work as expected now.

File: data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/arrive_in_country.jsonnet (hunk 1 above)
Comment (MebinAbraham) [reply data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/arrive_in_country.jsonnet]: Removed.

**Review by berroar — APPROVED** [review]
Routing all seems to be correct as per the trello card.

**Review by donnap573 — APPROVED** [review]

**Review by bambrp — APPROVED** [review]

**Pull request merged**